website/publish/on-premise_geo-ip-installation.rst

```diff
@@ -0,0 +1,58 @@
+===========================================
+Geo IP installation (on-premise database)
+===========================================
+
+Installation
+============
+
+.. warning:: Please note that the installation depends on your computer operating system and distribution. We will assume here that a Linux operating system is used.
+
+#. Install `geoip2 <https://pypi.org/project/geoip2/>`__ python library
+    .. code-block:: bash
+
+      pip install geoip2
+
+#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`__. You should end up with a file call ``GeoLite2-City.mmdb``
+#. Move the file to the folder ``/usr/share/GeoIP/``
+    .. code-block:: bash
+
+        mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
+
+#. Start the server
+
+.. note:: If for some reason you can't perform step 3 you can use the ``--geoip-db`` option instead.
```

Comment:
I would rephrase: "If you can't/don't want to locate the geoip database in `/usr/share/GeoIP/`, you can use the --geoip-db option of the Odoo cmd line interface" + add a link to the doc of the CLI